server/src/test/java/org/apache/druid/server/coordinator/duty/KillAuditLogTest.java

```diff
@@ -93,9 +94,16 @@ public void testConstructorFailIfInvalidPeriod()
         .withCoordinatorKillMaxSegments(10)
         .withCoordinatorKillIgnoreDurationToRetain(false)
         .build();
-    exception.expect(IllegalArgumentException.class);
-    exception.expectMessage("coordinator audit kill period must be >= druid.coordinator.period.metadataStoreManagementPeriod");
-    killAuditLog = new KillAuditLog(mockAuditManager, druidCoordinatorConfig);
+
+    final IllegalArgumentException exception = Assert.assertThrows(
+        IllegalArgumentException.class,
+        () -> killAuditLog = new KillAuditLog(mockAuditManager, druidCoordinatorConfig)
```

Comment:
why are we setting `killAuditLog`, couldn't this be simplified to 
```
Assert.assertThrows(
        IllegalArgumentException.class,
        () -> new KillAuditLog(mockAuditManager, druidCoordinatorConfig)
);
```

Reply:
Yeah, I would prefer it to be so, but it shows up as an inspection warning if you don't assign the return value to some variable.